README.md

```diff
@@ -1,68 +1,49 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Quiz web app.
 
-## Available Scripts
+## Description:
+This is a quiz web app helps users to check their knowledge of CYF study material.
+answer all the questions with multi choice answers and the result will be submitted afterwards.
 
-In the project directory, you can run:
+## Installation: 
+    1. Clone files into your local machine using - `git clone https://github.com/CodeYourFuture/project-quiz.git`.
+    2. Run npm install in your local terminal.
+    3. Run npm start to run the app or open `http://localhost:3000` to view it in the browser.
+    4. Add an origin remote pointing to the main CYF repo (to be able to pull updates) by using this code
+       - `git remote add origin https://github.com/CodeYourFuture/project-quiz.git`.
 
-### `yarn start`
+## Table of Contents: 
```

Comment:
I thought we could add the names of quizzes, for example : HTML, CSS,  etc.

Reply:
The readme/documentation should reflect the current state of the project, so since we don't have any quizzes right now then we can remove this. Once we have quizzes, I think it would make sense to list them in the description section since the list of quizzes is a description of what different quizzes you can do in the app